yoyodyne/models/transducer.py

```diff
@@ -465,23 +462,28 @@ def training_step(self, batch: data.PaddedBatch, batch_idx: int) -> Dict:
             on_step=False,
             on_epoch=True,
         )
-        return loss
 
-    def validation_step(self, batch: data.PaddedBatch, batch_idx: int) -> Dict:
+    def on_validation_epoch_start(self) -> None:
```

Comment:
Shouldn't this be inherited? Or does some class in its inheritance override this so we have to override it back?